testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -97,6 +104,160 @@ QUnit.testInActiveWindow('Form\'s inputs saves value on refresh', function(asser
     assert.deepEqual(formData, { name: 'test' }, 'value updates');
 });
 
+
+['xs', 'sm', 'md', 'lg'].forEach(screenSize => {
```

Comment:
Can we use "domAdapter.getDocumentElement()['clientWidth']" instead of 'screenSize/defaultScreenFactorFunc' ?

Reply:
Yep, added a new test to check default behavior of the 'defaultScreenFactorFunc'